capstone-backend/src/main/java/com/google/sps/servlets/MeetingServlet.java

```diff
@@ -0,0 +1,142 @@
+package com.google.sps.servlets;
+
+import com.google.api.client.auth.oauth2.BearerToken;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.auth.oauth2.TokenResponse;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventAttendee;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.api.services.calendar.model.EventReminder;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+
+import com.google.sps.data.Meeting;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/meetings")
+public class MeetingServlet extends HttpServlet {
+  private Firestore db;
+  private CollectionReference meetings;
+  private Gson gson;
+
+  public MeetingServlet() throws IOException {
+    db = Utility.getFirestoreDb();
+    meetings = db.collection("meetings");
+    gson = new Gson();
+  }
+
+  public static Credential createCredential(HttpTransport transport, JsonFactory jsonFactory,
+      TokenResponse tokenResponse) {
+    return new Credential(BearerToken.authorizationHeaderAccessMethod()).setFromTokenResponse(tokenResponse);
```

Comment:
nit: Can you format this line and those below to be within the 80 character limit?

Reply:
Sure! I think the character limit was 100, but regardless the below line is too long